### PR TITLE
Liveblogs - SpecialReport palette

### PR DESCRIPTION
**Pull request by OllysCoding**
Co-Authored-By: Joshua <joshuathomaslieberman@gmail.com>

## What does this change?

Update our palette to 'complete' support for special report liveblogs. 

> Note: In a few places we also changed the link colour to be darker for special reports, this was to try and get it to pass WCAG contrast guidelines!

## Why?

As our existing article palette is partially resused for liveblogs, we had a half supported half not supported type deal going on, so it makes more sense to complete the special report palette, than move all the way back to news styles!

## Screenshots

| type   | Before      | After      |
|-----------|-------------|------------|
| Live | ![before][] | ![after][] |
| Dead | ![before2][] | ![after2][] |

[before]: https://user-images.githubusercontent.com/3300789/168848579-0ea5ba25-ab05-4d30-9dc3-d60e9793a9b8.png
[after]: https://user-images.githubusercontent.com/3300789/168848168-a845a433-4c24-4555-9dac-064f813f0fed.png
[before2]: https://user-images.githubusercontent.com/3300789/168848678-3cd6d42f-ecc7-4146-af6a-2a55f72fe947.png
[after2]: https://user-images.githubusercontent.com/3300789/168848090-55a44366-e85a-427a-bf4a-59a6a87b729d.png

Example Articles to test with:
https://www.theguardian.com/news/live/2021/oct/05/pandora-papers-expose-eu-plans-on-tax-havens-as-absurd-meps-claim-live?dcr&live
https://www.theguardian.com/news/live/2021/oct/04/pandora-papers-live-sunak-says-hmrc-will-review-leaked-documents?dcr&live

### CODE REVIEW

**Comment by OllysCoding** [comment]
@HarryFischer I'd love if you could take a look at this!

File: dotcom-rendering/src/web/lib/decidePalette.ts
Comment (oliverlloyd) [inline dotcom-rendering/src/web/lib/decidePalette.ts:672]: This comment is now untrue (we _do_ support special reports, yay!)

File: dotcom-rendering/src/web/lib/decidePalette.ts
Comment (oliverlloyd) [inline dotcom-rendering/src/web/lib/decidePalette.ts:854]: Ditto

**Review by oliverlloyd — APPROVED** [review]
This is a beautiful PR 😍 . I love that we made the extra effort to add support for this even when there wasn't anyone shouting for it and I also love how simple the code changes are.

We just need to tidy up the comments to reflect these changes. Maybe it's easier at this point to just not have comments on this at all, it's arguably clear from the code what's happening? But maybe I'm biased as I already understand the issue - your call.

**Comment by oliverlloyd** [comment]
I should also point out that this PR obviously needs sign off from @HarryFischer before merging. I'm approving the code here, not the design

**Comment by HarryFischer** [comment]
Looks good, can we change a couple colours please

- body copy and key event links: specialReport.300 #3F464A 
(Looks like it's correct in the code but the colour is wrong in browser, unless I'm reading it wrong)

- page background (below header): specialReport.800 #EFF1F2

**Review by HarryFischer — APPROVED** [review]
👍

**Pull request merged**
